internal/build/ocpbuild/manager.go

```diff
@@ -43,9 +43,22 @@ func NewManager(
 }
 
 func (m *manager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
```

Comment:
WDYT about using something like
```go
	fieldOpt := client.MatchingFields(
		map[string]string{".status.phase": string(buildv1.BuildPhaseComplete)},
	)

	labelOpt := client.MatchingLabels(map[string]string{"key": "value"})

	if err := m.client.DeleteAllOf(ctx, &buildv1.Build{}, fieldOpt, labelOpt); err != nil {
		return err
	}
```

Reply:
actually, it is good, but then it means that we cannot know what we have deleted and we won't be able to log it (which is think it pretty beneficial for debugging). I suggest that we postpone your suggestion for a later commit, which will start from the upstream: we are going to change the GarbageCollect interface, since we don't really need owner parameter. We can discuss your suggestion in that PR. WDYT?